docs/requirements.txt

```diff
@@ -13,6 +13,7 @@ six==1.10.0
 smmap==0.9.0
 snowballstemmer==1.2.1
 sphinx==1.6.5
+sphinx-rtd-theme==0.2.4
```

Comment:
How come this was needed?

Reply:
I get the following error when I try to run ./docs/build.sh

```
Exception occurred:
  File "conf.py", line 15, in <module>
ImportError: No module named sphinx_rtd_theme
The full traceback has been saved in /var/folders/_s/vfl3wqps6q56jm6q7l3zz0j40000gp/T/sphinx-err-2RnUyU.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
``
I don't know where the dependency is coming from.

Reply:
Ah, it looks like I had an old version of sphinx installed globally. Once I uninstalled that, the error went away.